packages/e2e-test-utils-playwright/src/request-utils/posts.js

```diff
@@ -3,6 +3,27 @@
  *
  * @this {import('./index').RequestUtils}
  */
+
+export async function createNewPage( title, status = 'draft' ) {
```

Comment:
nit: What do you think about going with just `createPage`? I think "create" implies that it will be a new page so we can skip the "new" bit.